cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -17,109 +17,130 @@
 
 @pytest.mark.meta(blockers=[1273654])
 def test_assign_compute_enterprise(virtualcenter_provider):
+    app = get_or_create_current_appliance()
+    view = navigate_to(app.server, 'Chargeback')
+
     enterprise = cb.Assign(
         assign_to="The Enterprise",
         selections={
-            "Enterprise": "Default"
+            'Enterprise': {'Rate': 'Default'}
         })
     enterprise.computeassign()
 
-    flash.assert_message_match('Rate Assignments saved')
     # Assert that the selection made is listed on the UI
-    selected_option = sel.text(
-        cb.assign_form.selections.select_by_name("Enterprise").first_selected_option)
+    assign_view = view.browser.create_view(AssignmentsView)
+    row = assign_view.selections.row(name='Enterprise')
+    selected_option = row.rate.widget.selected_option
     assert selected_option == "Default", 'Selection does not match'
 
 
 def test_assign_compute_provider(virtualcenter_provider):
+    app = get_or_create_current_appliance()
```

Comment:
**REQUIRED**
I propose to not get the appliance object, but import Server class:
```python
from cfme.base import Server
...
view = navigate_to(Server, 'Chargeback')
```